dockfix.go

```diff
@@ -127,3 +150,23 @@ func RemoveContainer(c Container) error {
 	containerFileName := c.name + ".container"
 	return os.Remove(containerFileName)
 }
+
+func imageExists(baseImage string) (bool, error) {
+	dc, _ := NewClient()
+	imageName := strings.Split(baseImage, ":")[0]
+	images, err := dc.ListImages(docker.ListImagesOptions{Filter: imageName})
+	if err != nil {
+		return false, err
+	}
+
+	if len(images) > 0 {
```

Comment:
This isn't really needed. If length is 0, the range loop just won't do anything.